src/coord/src/session/vars.rs

```diff
@@ -100,6 +100,12 @@ const INTEGER_DATETIMES: ServerVar<bool> = ServerVar {
     description: "Reports whether the server uses 64-bit-integer dates and times (PostgreSQL).",
 };
 
+const INTERVALSTYLE: ServerVar<str> = ServerVar {
```

Comment:
nit: we do `DATE_STYLE` for `DateStyle`, so should do `INTERVAL_STYLE` to match.

Reply:
Done!